### PR TITLE
Refactor/dcox/20190216

### PR DESCRIPTION
**Pull request by DCox2016**
Made a dashboard view component and moved amplify-authenticaticator from app.compnent.html to dashboard.component.html.

### CODE REVIEW

File: src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'ata-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.css']
+})
+export class DashboardComponent implements OnInit {
+
+  constructor() { }
```
Comment (cpyle0819) [inline src/app/dashboard/dashboard.component.ts]: This isn't required if it has no body.

File: src/app/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  ngOnInit() {
```
Comment (cpyle0819) [inline src/app/dashboard/dashboard.component.ts]: same here

File: src/app/login/login.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<span *ngIf="signedIn$ | async">Hello, {{ (user$ | async).username }}</span>
```
Comment (cpyle0819) [inline src/app/login/login.component.html]: You have `signedIn$` named as if it were an observable, but it's not.

File: src/app/login/login.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  styleUrls: ['login.component.css']
+})
+export class LoginComponent {
+  signedIn: boolean;
+  user: any;
+  greeting: string;
+
+    constructor( private amplifyService: AmplifyService ) {
+   
+    this.amplifyService.authStateChange$
```
Comment (cpyle0819) [inline src/app/login/login.component.ts]: I thought we stepped through refactoring this so `signedIn` became an observable.

Comment (DCox2016) [reply src/app/login/login.component.ts]: The refactoring kept the amplifyService from working properly so I switched back.

File: src/app/dashboard/dashboard.component.ts (hunk 1 above)
Comment (DCox2016) [reply src/app/dashboard/dashboard.component.ts]: removed

File: src/app/dashboard/dashboard.component.ts (hunk 2 above)
Comment (DCox2016) [reply src/app/dashboard/dashboard.component.ts]: removed

File: src/app/login/login.component.html (hunk 3 above)
Comment (DCox2016) [reply src/app/login/login.component.html]: Was not needed to display log in. Has been removed.

**Comment by cpyle0819** [comment]
PR title doesn't match standards.

**Pull request merged**